src/components/HURUmap/IndicatorTitle/Share.js

```diff
@@ -27,8 +40,8 @@ function Share({ title, geoCode, indicatorId, view, isCompare, ...props }) {
     },
     { name: "WhatsApp", props: { quote: title } },
     { name: "Email", props: { subject: title } },
+    { name: "Copy" },
```

Comment:
Is there a way of setting a different tooltip? I think `Copy URL` will be a more descriptive tools since copy doesn't tell you what exactly have you copied.

Reply:
@kilemensi Yap! Changed it to `CopyUrl` in the cms. I think that should work

Reply:
@kilemensi Changed the names for both the social links in the cms and the chart to `CopyUrl`